packages/components/src/lib/controllers/ui-controller.js

```diff
@@ -1920,7 +1920,9 @@ NB this is Prototype functionality
     if (this.enableMouseMoveEvent()) {
       const eventParams = {
         mouseMoveDelay: this.featureOptions.items.mouseMoveDelay.currentValue,
-        mouseMoveAccuracy: this.featureOptions.items.mouseMoveAccuracy.currentValue
+        mouseMoveAccuracy: this.featureOptions.items.mouseMoveAccuracy.currentValue,
+        enableMouseMoveLimitedByIdCheck: this.featureOptions.items.enableMouseMoveLimitedByIdCheck.currentValue,
```

Comment:
@irina060981 I am still thinking that this would apply to ANY mousemove event, not just one on Google docs. Are you sure that it will only be limited by id for the google docs page? enableMouseMoveEvent will return true if mouse move is enabled and the platform is not google docs...

Reply:
yes, you are right! I am sorry, I thoght how to turn it to more wide variants in future and left it this way.
I will limit to google docs